### PR TITLE
Hot fix for SB-94

### PR DESCRIPTION
**Pull request by StudentCZ**
Henry and I pair program to fix the bug that SB-94 causes when merging. 

Screen Shot 2023-01-31 at 6 57 24 PM

To this
Screen Shot 2023-01-31 at 6 58 08 PM

Error is showing no routesdetail because the backend and frontend endpoints are not connected for the routes. Other than that everything is working so far.

### CODE REVIEW

File: src/utils/axios.js [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
+    Accept: 'application/json',
   },
 });
 
 export const postHeartBeat = async (data) => {
   try {
-    const res = await apiRequest.post("/heartbeat", data);
-    console.log(res);
+    const res = await apiRequest.post('/heartbeat', data);
+    console.log('successfully sent');
```
Comment (yhus) [inline src/utils/axios.js:18]: Heartbeat was successfully sent?

**Review by yhus — COMMENTED** [review]
This looks good, pending resolving the ESLint warnings.

For future PRs, please make sure to split up styling changes with actual logic/code changes. It seems like most of the changes in this PR were styling related, which makes reviewing very difficult since we'd need to carefully scan line by line for actual code changes (not much point in reviewing styling changes line by line...)

**Review by FioraDeMarco — APPROVED** [review]
I think it should be changed to heartbeat was successfully sent, but other than that, it's good.

**Review by FioraDeMarco — APPROVED** [review]

File: src/utils/axios.js (hunk 1 above)
Comment (StudentCZ) [reply src/utils/axios.js:18]: ok I changed the message to make it more clearer.

**Review by nicolepa — APPROVED** [review]
at high level, the changes made to resolve conflict that was identified while trying to merge SB94 look sound - team member from data/BE

**Pull request merged**